attributes/default.rb

```diff
@@ -1,3 +1,4 @@
 default['optoro_memcached']['log_dir'] = '/var/log/memcached'
 default['optoro_memcached']['log_path'] = "#{node['optoro_memcached']['log_dir']}/memcached.log"
 default['optoro_memcached']['mem_size_mb'] = '512'
+default['monit']['address'] = 'localhost'
```

Comment:
Its the base domain, but it is populated via the original monit cookbook. So we have to override it anyway with our information.